packages/shared/src/components/utilities/common.tsx

```diff
@@ -111,7 +111,7 @@ export const FeedPage = classed(
 export const FeedPageLayoutV1 = classed(
   BasePageContainer,
   pageContainerClassNames,
-  'pt-10 tablet:px-6 !ml-auto !px-0',
+  'pt-10 tablet:px-6 !ml-auto !px-0 !max-w-full',
```

Comment:
Why is this `!max-w-full` needed?

Reply:
We need it, or else the tablet mode would result differently. From Figma, the tablet mode should consume the whole screen width.

Without the max width:
![Screenshot 2024-01-04 at 7 03 50 PM](https://github.com/dailydotdev/apps/assets/13744167/ffcedb4b-774f-41b5-9c6a-11b3e86f9b02)

Figma:
![Screenshot 2024-01-04 at 7 04 25 PM](https://github.com/dailydotdev/apps/assets/13744167/18594edb-f852-4a59-a3f0-0f0add5b220d)

Reply:
Are we sure we want that from 656px to 1020px? 
<img width="1001" alt="Screenshot 2024-01-04 at 12 23 05 PM" src="https://github.com/dailydotdev/apps/assets/36197304/1bb5118e-bd59-419a-af50-9a01c718e5c2">
Looks weird to me in larger screens.

I think when setting the breakpoints I had taken this as, it should look like this from 656px and downwards, not upwards. But might be me mistaken. 
Shall we check with design to make sure?

Reply:
Yeah, this case will have to be checked with the design team.

Reply:
Cool, added here https://dailydotdev.slack.com/archives/C061T644XV3/p1704368093654569, lets see what they say and move on accordingly

Reply:
Nice, we can leave this then, it was my misunderstanding 🙌

Reply:
After checking with design, we want the feed to have a max width of 680px, which it already has, the fullwidth will come naturally in screens smaller then 680px.